### DISCUSSION TITLE
Strange symbols appearing when accessing the website

### DISCUSSION THREAD

**Discussion by gcarpi** [vercel/community#3145] [Help] [answered]
### Summary

Some strange symbols appear when accessing the site, and this is only happening with some users of the system. Apparently it has something to do with the user's network, because when changing the network the site works again. 

The error is apparently coming from the file: https://vercel.live/_next-live/feedback/feedback.js

![Screenshot 2023-07-06 at 12 43 56](https://github.com/vercel/community/assets/19765074/e04a751f-b4e3-41df-81f5-6116e3d5092a)

![image](https://github.com/vercel/community/assets/19765074/85ac581c-0062-4c7b-a9d6-e9ab22e23007)

### Example

https://hub.modular.dev.br

### Steps to Reproduce

Access the [link](https://hub.modular.dev.br/) and maybe the error will happen to you.

**Comment by eliasassuncao** [comment]
up

**Comment by DouglasCalora** [comment]
up

**Comment by amyegan** [comment]
I'm not able to reproduce the error with that site. Do you have more information about what network differences seem to cause this? For example, it may be related to speed, region, or ISP.

**Reply by gcarpi** [reply]
We were able to solve the problem by disabling the "Preview Comments" option in the project settings. Apparently there is a problem related to this functionality.

![Screenshot 2023-07-06 at 18 29 21](https://github.com/vercel/community/assets/19765074/dfb4798a-8ed1-40bf-a12b-5ee7ae18c8e0)

**Reply by amyegan** [reply, marked as answer]
🤔 Based on some similar reports I found, this may have been related to a recently resolved [Cloudflare proxy issue](https://www.cloudflarestatus.com/incidents/2gslw685z9w2). 

I'm glad that it's working for you now! 🙌
